scala-thoughts.md

```diff
@@ -0,0 +1,37 @@
+### What's easy to do in Scala? What's not?
+* I was impressed with the variety of methods on lists! I used map and filter, because those
+were methods that I learned in my CS52 class last semester. 
+One of Scala's greatest strengths is how concise it can be,
+after you learn how to properly chain methods. 
+
+### What is/are your favorite language design choice(s) that the designers of Scala made? Why?
+* To be honest, I found this assignment to be quite difficult... but here are some of the cool things that I observed.
+* I don't know whether I like this, but I think it's interesting that Scala is designed so that a "phrase of code" doesn't need
+to end in a semicolon. I think it's nice that having it doesn't hurt. Maybe this is because Scala was designed to cater to the 

```

Comment:
I think Scala is really good at giving users lots of options. I find that people complain about semicolon/no semicolon when comparing languages like Java vs. JavaScript vs. Python. So Scala can easily prevent complaints by providing both options.